catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -337,6 +344,12 @@
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(Validation.VALIDATORS,
+                true /* indexed */,
```

Comment:
For the case where a user might want to search on validators that have passed/failed